### ISSUE TITLE
[Bug] cleaning drone has no hitbox now?

### ISSUE THREAD

**Issue by JohnMetroid** [pseudopulse/Moonswept#3]
Hey there,

Prior to 1.0.0, the cleaning drone could be killed in conjunction with EverythingCanDie.  However, upon updating to Moonswept 1.0.0 and confirming the cleaning drone was still hittable, it doesn't appear to be.  The only thing I can guess is the hitbox has been removed/adjusted/disabled on Moonswept's end?

![image](https://github.com/user-attachments/assets/caba0ec8-ca26-4d62-b403-3fe5f1ec62c4)

**Comment by Smxrez** [comment]
yeah the hitbox position was accidentally messed up it will be fixed in the next update
